Dockerfile

```diff
@@ -3,14 +3,14 @@ MAINTAINER colab oratory
 
 # the folder is called /vagrant for compatability with vagrant... should
 # probably be called /oratory
-RUN mkdir -p /vagrant/oratory/ /vagrant/lib
+RUN mkdir -p /vagrant/oratory/ /opt/node/lib/
 
-WORKDIR /vagrant/lib/
-ADD ./oratory/package.json /vagrant/lib/
+WORKDIR /opt/node/lib/
+ADD ./oratory/package.json /opt/node/lib/
 RUN npm install && \
     npm install -g supervisor
-ENV NODE_PATH /vagrant/lib/node_modules:$NODE_PATH
+ENV NODE_PATH /opt/node/lib/node_modules:$NODE_PATH
 
 WORKDIR /vagrant/oratory/
 EXPOSE 80
-
+CMD touch users.js && supervisor server.js
```

Comment:
This is meant for dev, not prod. I vote leaving it as supervisor

On Thu, Oct 15, 2015, 4:20 PM lju <notifications@github.com> wrote:

> In Dockerfile
> <https://github.com/colabprojects/oratory/pull/74#discussion_r42174850>:
>
> >
> >  WORKDIR /vagrant/oratory/
> >  EXPOSE 80
> > -
> > +CMD touch users.js && supervisor server.js
>
> I think supervisor is intended for development and not production. I'm a
> fan of switching this to forever, or if docker can be environment aware
> then doing either depending.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/colabprojects/oratory/pull/74/files#r42174850>.
>
-- 
justin murray
addpcs, LLC  |  *software person*
murrayju@addpcs.com